src/main/scripts/package.sh

```diff
@@ -167,6 +167,7 @@ if [[ "$platform" == "mac" ]]; then
     fi
 
     if [[ "$properties_mac_sign" == "1" ]]; then
+        export JPACKAGE_MAC_SIGN="1"
```

Comment:
Toto tu ma byt na tvrdo?

Reply:
ano, jedine ine co by tam davalo zmysel je `"$properties_mac_sign"` ale to je tiez "1"